tools/run_onnx.cc

```diff
@@ -534,6 +537,8 @@ void RunMain(const std::vector<std::string>& argv) {
     args.add<double>("atol", '\0', "atol of AllClose", false, 1e-6);
     args.add("equal_nan", '\0', "Treats NaN equal");
     args.add("no_catch", '\0', "Do not catch the exception in ChxVM for better GDB experience");
+    args.add("no_strict_check", '\0', "Disable strict check of node output");
+    args.add("show_diff", '\0', "Show diff of strict check without stopping");
```

Comment:
How about `always_show_diff`? Also, how about "Show diff even when value check is skipped" for the help message?